pyproject.toml

```diff
@@ -6,7 +6,7 @@ name = "quantus"
 # https://www.python.org/dev/peps/pep-0440/
 description = "A metrics toolkit to evaluate neural network explanations."
 readme = "README.md"
```

Comment:
remove the flit ref and replace with documentation for the new build system

https://flit.pypa.io/en/stable/pyproject_toml.html

Reply:
I've reverted back to `flit`.